dex/models/trades/sei/platforms/jelly_swap_sei_base_trades.sql

```diff
@@ -0,0 +1,22 @@
+{{
+    config(
+        schema = 'jelly_swap_v2_sei',
```

Comment:
```suggestion
        schema = 'jelly_swap_sei',
```

Reply:
should we remove `v2` to match file name? or do the opposite and keep version but add to files?